### ISSUE TITLE
About the Epoch

### ISSUE THREAD

**Issue by XDZxdz1** [pykt-team/pykt-toolkit#155]
Hello, sorry to bother you again. May I ask if all models have a training Epoch of 200？

**Comment by sonyawong** [comment]
> Hello, sorry to bother you again. May I ask if all models have a training Epoch of 200？

Yes, The training epoch of each model is set to 200. However,  we employ early stop hence the total training epoch of each model is different.
